src/plugins/walletConnectToDapps/components/header/WalletConnectToDappsHeaderButton.tsx

```diff
@@ -0,0 +1,60 @@
+import { ChevronDownIcon, ChevronRightIcon } from '@chakra-ui/icons'
+import { Menu, MenuButton, MenuList } from '@chakra-ui/menu'
+import { Button } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { WalletConnectIcon } from 'components/Icons/WalletConnectIcon'
+import { RawText } from 'components/Text'
+
+import { DappAvatar } from './DappAvatar'
+import { DappHeaderMenuSummary } from './DappHeaderMenuSummary'
+
+export const WalletConnectToDappsHeaderButton = () => {
+  const dapp: any = {
+    name: 'Uniswap',
+    link: 'app.uniswap.org',
+    image:
+      'https://rawcdn.githack.com/trustwallet/assets/master/blockchains/ethereum/assets/0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984/logo.png',
+    chainId: 1,
+    connected: true,
+    address: '0x123321123123321',
+  }
+  const translate = useTranslate()
+
+  if (!dapp) {
+    return (
+      <Button leftIcon={<WalletConnectIcon />} rightIcon={<ChevronRightIcon />}>
+        {translate('plugins.walletConnectToDapps.header.connectDapp')}
+      </Button>
+    )
+  }
+
+  return (
+    <Menu autoSelect={false}>
+      <MenuButton
+        as={Button}
+        leftIcon={
+          <DappAvatar
+            name={dapp.name}
+            image={dapp.image}
+            connected={dapp.connected}
+            size={6}
+            connectedDotSize={2}
+            borderWidth={1}
+          />
+        }
+        rightIcon={<ChevronDownIcon />}
+        width={{ base: 'full', md: 'auto' }}
+        textAlign='left'
+      >
+        {/* TODO: when setting "flex: unset" or "flex-shrink: none" to the Button content parent, overflow isn't a problem */}
```

Comment:
![image](https://user-images.githubusercontent.com/108498243/190268857-4b566ae9-02d2-4259-84db-de09214bee73.png)

For some reason the text doesn't fill its height here properly. I've tried with toggling different flex CSS props, but can't figure out what's going on. Any idea?
![image](https://user-images.githubusercontent.com/108498243/190268983-09bb0a0a-52c7-48a8-a26c-f826335f219e.png)

The TODO hax isn't really a solution, it just creates new (less bad) problems

Reply:
@reallybeard halp?